tox.ini

```diff
@@ -24,12 +24,13 @@ deps =
     lockfile==0.10.2
     coverage
     mock==1.0.1
-    django14: Django==1.4.22
```

Comment:
what's the process for bumping the point versions? should we consider having the unbounded in the allowed failures section?